vhdbuilder/release-notes/AKSUbuntu/gen1/1604/2021.02.24.txt

```diff
@@ -0,0 +1,218 @@
+  - mcr.microsoft.com/oss/kubernetes/pause:1.2.0
+  - mcr.microsoft.com/oss/kubernetes/pause:1.3.1
+  - mcr.microsoft.com/oss/kubernetes/pause:1.4.0
+  - mcr.microsoft.com/oss/kubernetes/coredns:1.6.6
+  - mcr.microsoft.com/oss/kubernetes/coredns:1.6.5
+  - mcr.microsoft.com/oss/kubernetes/coredns:1.5.0
+  - mcr.microsoft.com/oss/kubernetes/coredns:1.3.1
+  - mcr.microsoft.com/oss/kubernetes/coredns:1.2.6
+  - mcr.microsoft.com/containernetworking/networkmonitor:v1.1.8
+  - mcr.microsoft.com/containernetworking/networkmonitor:v0.0.7
+  - mcr.microsoft.com/containernetworking/networkmonitor:v0.0.6
+  - mcr.microsoft.com/containernetworking/azure-npm:v1.2.3
+  - mcr.microsoft.com/containernetworking/azure-npm:v1.2.2_hotfix
+  - mcr.microsoft.com/containernetworking/azure-npm:v1.2.1
+  - mcr.microsoft.com/containernetworking/azure-npm:v1.1.8
+  - mcr.microsoft.com/containernetworking/azure-npm:v1.1.7
+  - mcr.microsoft.com/containernetworking/azure-vnet-telemetry:v1.0.30
+  - nvidia/k8s-device-plugin:1.11
+  - nvidia/k8s-device-plugin:1.10
+  - mcr.microsoft.com/aks/hcp/hcp-tunnel-front:v1.9.2-v3.0.18
+  - mcr.microsoft.com/aks/hcp/hcp-tunnel-front:v1.9.2-v3.0.19
+  - mcr.microsoft.com/aks/hcp/hcp-tunnel-front:v1.9.2-v3.0.20
+  - mcr.microsoft.com/oss/kubernetes/apiserver-network-proxy/agent:v0.0.13
+  - mcr.microsoft.com/aks/hcp/tunnel-openvpn:1.0.8
+  - mcr.microsoft.com/aks/hcp/tunnel-openvpn:1.0.10
+  - mcr.microsoft.com/aks/hcp/tunnel-openvpn:1.0.11
+  - mcr.microsoft.com/aks/hcp/kube-svc-redirect:v1.0.7
+  - mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod10272020
+  - mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod11092020
+  - mcr.microsoft.com/azuremonitor/containerinsights/ciprod:ciprod01112021
+  - mcr.microsoft.com/oss/calico/cni:v3.8.9.1
+  - mcr.microsoft.com/oss/calico/cni:v3.8.9.2
+  - mcr.microsoft.com/oss/calico/node:v3.17.2
+  - mcr.microsoft.com/oss/calico/node:v3.8.9.1
+  - mcr.microsoft.com/oss/calico/node:v3.8.9.2
+  - mcr.microsoft.com/oss/calico/typha:v3.17.2
+  - mcr.microsoft.com/oss/calico/typha:v3.8.9
+  - mcr.microsoft.com/oss/calico/kube-controllers:v3.17.2
+  - mcr.microsoft.com/oss/tigera/kube-controllers:v1.13.5
+  - mcr.microsoft.com/oss/calico/pod2daemon-flexvol:v3.8.9
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.3.0_v0.0.5
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.7.1
+  - mcr.microsoft.com/oss/kubernetes/autoscaler/cluster-proportional-autoscaler:1.7.1-hotfix.20200403
+  - mcr.microsoft.com/k8s/flexvolume/keyvault-flexvolume:v0.0.13
+  - mcr.microsoft.com/k8s/flexvolume/blobfuse-flexvolume:1.0.15
+  - mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.2
+  - mcr.microsoft.com/oss/kubernetes/ip-masq-agent:v2.5.0.3
+  - nginx:1.13.12-alpine
+  - mcr.microsoft.com/k8s/kms/keyvault:v0.0.9
+-rwxr-xr-x 1 root root 150582248 Jun  1  2020 /usr/local/bin/kubelet-1.17.3
```

Comment:
what's these?

Reply:
https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/install-dependencies.sh#L288
seems the logging is in mid. I'll send a PR to move it to a better position